content/introduction/introduction_notation.rst

```diff
@@ -38,11 +37,11 @@ Integrals
     .. math::
         \oint_S \mathbf{f} \cdot \mathbf{da} = \oint_S \mathbf{f} \cdot \mathbf{\hat{n}} ~da
 
-- Integrating a vector function over a line: 
+- Integrating a vector function over a curve: 
```

Comment:
This kind of integrals are typically known as line integrals.  However, typically we integrate over curves.